pkg/server/server.go

```diff
@@ -313,7 +313,7 @@ func initAdmission(ctx context.Context, vConfig *rest.Config) (admission.Interfa
 		&emptyConfigProvider{},
 		admission.PluginInitializers{
 			webhookinit.NewPluginInitializer(authInfoResolverWrapper, serviceResolver),
-			initializer.New(vClient, kubeInformerFactory, nil, nil, nil),
+			initializer.New(vClient, nil, kubeInformerFactory, nil, nil, nil),
```

Comment:
The second parameter is for now used only in the "[celAdmissionPlugin](https://github.com/kubernetes/kubernetes/blob/97ab147537f6999249c241e0896245bd961c32ae/staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission.go#LL79C27-L79C27)", but we only register the "mutating" and "validating" plugins.

Reply:
ok